Benchmarks/Package.swift

```diff
@@ -0,0 +1,69 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+import class Foundation.FileManager
+import class Foundation.ProcessInfo
+
+// Note: the JAVA_HOME environment variable must be set to point to where
```

Comment:
I unfortunately had to copy this out of `/Package.swift` since it's relied upon for setting include paths in `JavaApiCallBenchmarks`.

Reply:
yea we have to copy this around, thx!